spec/timezone.html

```diff
@@ -134,7 +134,18 @@ <h1>Temporal.TimeZone.prototype.getOffsetNanosecondsFor ( _absolute_ )</h1>
         1. Let _timeZone_ be the *this* value.
         1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
         1. Perform ? RequireInternalSlot(_absolute_, [[InitializedTemporalAbsolute]]).
-        1. Return ? GetTimeZoneOffsetNanoseconds(_absolute_.[[Nanoseconds]], _timeZone_.[[Identifier]]).
+        1. Let _offset_ be ! ParseOffsetNanoseconds(_timeZone_.[[Identifier]]).
```

Comment:
This is the way it's implemented in the polyfill, but I wonder if we could consider having an internal slot in the spec to distinguish between whether a time zone object represents an offset or named time zone.

Reply:
Let's wait and see if we end up keeping offset time zones.